### PR TITLE
Adding Endpoints

### PR DESCRIPTION
**Pull request by ms-alguerra**

### CODE REVIEW

**Comment by vladca** [comment]
Why did you downgrade the Authentication package?

File: src/ResourceManager/TrafficManager/Commands.TrafficManager2/Models/TrafficManagerEndpoint.cs [hunk 1]
```diff
@@ -16,25 +16,19 @@ namespace Microsoft.Azure.Commands.TrafficManager.Models
 {
     using System;
     using Microsoft.Azure.Commands.TrafficManager.Utilities;
+    using Microsoft.Azure.Management.TrafficManager.Models;
 
-    public class Endpoint
+    public class TrafficManagerEndpoint
```
Comment (vladca) [inline src/ResourceManager/TrafficManager/Commands.TrafficManager2/Models/TrafficManagerEndpoint.cs:21]: Is this a breaking change from the previous version?

**Comment by vladca** [comment]
I don't see anything for Add-TrafficManagerEndpointConfig. Don't you need to add TargetResourceId and make Target optional? Or have you and Jonathan made a deliberate call to not support AzureEndpoints in the new cmdlet?

File: src/ResourceManager/TrafficManager/Commands.TrafficManager2/Models/TrafficManagerEndpoint.cs (hunk 1 above)
Comment (ms-alguerra) [inline src/ResourceManager/TrafficManager/Commands.TrafficManager2/Models/TrafficManagerEndpoint.cs:21]: That is a fair question. However I think the type is transparent for the users. They used to handle endpoints with the Endpoint property of TrafficManagerProfile type. Plus, I have only added new properties. Therefore it should not be. Correct me if I am wrong.

**Pull request merged**